src/pages/authPage/signup.js

```diff
@@ -0,0 +1,50 @@
+import { Link } from "react-router-dom";
+import { Navbar } from "../../components";
+import "./auth.css";
+export function SignUp() {
+  return (
+    <div>
+      <Navbar />
+      <main>
+        <div className="form-container">
+          <div className="form-info">
+            <p className="heading-x-lg text-center">Join-us</p>
+            <div className="form-input">
+              <div className="input-box">
+                <label className="text-medium">Full name</label>
+                <input type="text" placeholder="Monkey D luffy" />
+              </div>
+              <div className="input-box">
+                <label className="text-medium">Email address</label>
+                <input type="text" placeholder="demo@yahoo.com" />
+              </div>
+              <div className="input-box">
+                <label className="text-medium">Password</label>
+                <input type="text" placeholder="*******" />
+              </div>
+              <div className="space-between forget-pass">
+                <span>
+                  <input type="checkbox" />
```

Comment:
Whenever any checkbox or radio button is involved, you can add the input inside the label so that checks and unchecks work even when the user clicks on the label.

Reply:
its resolve now